lib/pubid/itu/renderer/base.rb

```diff
@@ -7,7 +7,17 @@ def render(**args)
     end
 
     def render_identifier(params)
-      "%{publisher}-%{sector} %{type}%{series}%{number}%{part}" % params
+      "%{publisher}-%{sector} %{type}%{series}%{number}%{part}%{month}%{year}" % params
```

Comment:
Why not make `month` and `year` as a "date"? The separate rendering of month and year is very strange.

Reply:
agree! fixed.